scanner/scanner_service.py

```
@@ -0,0 +1,238 @@
+        while self.is_running:
+            try:
+                audio_list = self.fetch_audio_batch()
+
+                for audio in audio_list:
+                    if self.should_play_audio(audio):
+                        self.audio_queue.put(audio)
+                        self._notify_listeners('new_transmission', audio.to_dict())
+
+                time.sleep(fetch_interval)
+
+            except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)